enterprise/cli/dbcrypt_rotate.go

```diff
@@ -0,0 +1,131 @@
+//go:build !slim
+
+package cli
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+
+	"cdr.dev/slog"
+	"cdr.dev/slog/sloggers/sloghuman"
+
+	"github.com/coder/coder/v2/cli"
+	"github.com/coder/coder/v2/cli/clibase"
+	"github.com/coder/coder/v2/coderd/database"
+	"github.com/coder/coder/v2/codersdk"
+	"github.com/coder/coder/v2/enterprise/dbcrypt"
+
+	"golang.org/x/xerrors"
+)
+
+func (*RootCmd) dbcryptRotate() *clibase.Cmd {
+	var (
+		vals = new(codersdk.DeploymentValues)
+		opts = vals.Options()
+	)
+	cmd := &clibase.Cmd{
+		Use:   "dbcrypt-rotate --postgres-url <postgres_url> --external-token-encryption-keys <new-key>,<old-key>",
```

Comment:
running this from the CLI that has to have direct DB access is kind of obnoxious from an operator's standpoint.

What about making it an API call you could make to an appropriately-keyed Coderd?  We could still have the CLI command, but you could run it from anywhere with Coder access.  We could also consider a button on the front end at some later date.

Reply:
While this would be a nice usability improvement, it has precedence already with the `reset-password` CLI command. It's a good candidate for a follow-up PR though!